src/ntfs.rs

```diff
@@ -253,6 +256,60 @@ fn manage_feed_infos(collections: &mut Collections, path: &path::Path) {
     )
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone)]
+struct CommentLink {
+    object_id: String,
+    object_type: String,
+    comment_id: String,
+}
+
+fn insert_comment_link<T>(collection: &mut Collection<T>, comment_link: CommentLink)
+where
+    T: CommentLinks + Id<T>,
+{
+    let idx = match collection.get_idx(&comment_link.object_id) {
+        Some(idx) => idx,
+        None => {
+            error!(
+                "comment_links.txt: object_type={} object_id={} not found",
+                comment_link.object_type, comment_link.object_id
+            );
+            return;
+        }
+    };
+    collection
+        .index_mut(idx)
+        .comment_links_mut()
+        .push(comment_link.comment_id);
+}
+
+fn manage_comments(collections: &mut Collections, path: &path::Path) {
+    if csv::Reader::from_path(path.join("comments.txt")).is_ok() {
+        info!("Reading comments.txt");
+        collections.comments = make_collection(path, "comments.txt");
```

Comment:
better to have an optional version of make_collection, that try to create a Read, and then, if successful, create a csv::Reader::from_reader. Creating a csv::Reader for cheecking the presence of the file is a bit overkill.

Reply:
a make_collection_from_reader can be great

Reply:
or I can do `if path.join("comments.txt").exists() {`

Reply:
looks good, yes.

Reply:
even better: `is_file`

Reply:
maybe not, as you wish.